libs/ledger-live-common/src/reconciliation.ts

```diff
@@ -388,6 +389,15 @@ export function patchAccount(
     changed = true;
   }
 
+  if (
+    updatedRaw.celoResources &&
+    // @ts-expect-error check if this is valid for deep equal check
+    account.celoResources !== updatedRaw.celoResources
```

Comment:
@pawelnguyen - I believe `account.celoResources !== updatedRaw.celoResources` will always return false since they are different object instances.  You can use `lodash/isEqual` to fix this.

```
import isEqual from 'lodash/isEqual;

if (updatedRaw.celoResources && !isEqual(account.celoResources, updatedRaw.celoResources)) {
```

Reply:
that's a copy paste from other networks, I'll check if isEqual works. Good idea 👍